.github/workflows/workflow.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           sudo apt-get update --fix-missing
           sudo apt-get autoremove
           sudo apt-get autoclean
-          pip install tomte[tox]==0.1.5
+          pip install tomte[tox]==0.2.2
```

Comment:
make ci-requirements.txt and use `pip install -r` to make version edit easier?

Reply:
Good idea. Will open a Trello ticket